src/main/java/com/zendesk/maxwell/schema/ddl/DatabaseDrop.java

```diff
@@ -0,0 +1,28 @@
+package com.zendesk.maxwell.schema.ddl;
+
+import com.zendesk.maxwell.schema.Database;
+import com.zendesk.maxwell.schema.Schema;
+
+public class DatabaseDrop extends SchemaChange {
+	public String dbName;
+	public boolean ifExists;
+
+	public DatabaseDrop(String dbName, boolean ifExists) {
+		this.dbName = dbName;
+		this.ifExists = ifExists;
+	}
+
+	@Override
+	public Schema apply(Schema originalSchema) throws SchemaSyncError {
+		Schema newSchema = originalSchema.copy();
+		Database database = findDatabase(newSchema, dbName, ifExists);
```

Comment:
nitpic: The interchanging usage of findDatabase within SchemaChange and within Schema is confusing.